Platform/CommonBoardPkg/AcpiTables/Dbg2/Dbg2.act

```diff
@@ -0,0 +1,93 @@
+/** @file
+  This file contains a structure definition for the ACPI 2.0 Multiple APIC
+  Description Table (MADT).  Any changes to the MADT table require updating the
```

Comment:
MADT?

Reply:
and do we need add this file to CFL/APL build?

Reply:
Corrected the headers in new patch. Yes, platform needs to include it in AcpiTables.inf, if required.